src/components/MapAvailableDataDate.vue

```diff
@@ -17,8 +17,8 @@
             };
         },
         mounted() {
-            // If we can get a URL, try to get the date the model data was received and add it to the component data
-            if (process.env.VUE_APP_DATA_DATE_URL) {
+            // If we can get a URL, try to get the date the water storage model data was received and add it to the component data
+            if (process.env.VUE_APP_DATA_DATE_URL && this.$route.name==='waterStorage') {
```

Comment:
not sure if that's a trashy way to get to that info but it worked, same for next section